src/parsers/perf/perfparser.cpp

```diff
@@ -33,14 +33,11 @@
 #include <QBuffer>
 #include <QDataStream>
 #include <QFileInfo>
-#include <QLoggingCategory>
```

Comment:
you did the conflict resolution in the wrong direction here. please undo all changes to perfparser.cpp